<!DOCTYPE html>
<head>
<meta name="viewport" content="width=device-width" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<style>
@page {
	size: A4;
	margin: 20px;
}

.report {
	font-family: Arial, sans-serif;
	background: #fff;
	max-width: 800px;
	margin: 20px auto;
}

.report h1 {
	font-weight: 300;
	text-align: center;
	margin: 0 0 0.5rem 0;
	padding: 10px;
	font-size: 20px;
	border: 1px solid #e7e5e5;
	text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
}

h1.clinical,
h2.clinical {
	background: #fff2cc;
}

h3 {
	font-size: 0.9rem;
	margin-left: 1rem;
}

h1.technical {
	background: #bfbfbf;
}

h2.public_health {
	background: #bdd7ee
}

.info-grid {
	display: flex;
	flex-wrap: wrap;
	justify-content: space-between;
	gap: 12px;
	margin-bottom: 0.5rem;
}

.info-grid>div {
	flex: 1 1 auto;
	padding: 0 0.5rem;

}

.info-grid>div.bordered {
	border: 1px solid #e7e6e6
}

.section-title {
	padding: 6px 12px;
	margin: 0;
	border: 1px solid #e7e5e5;
	font-size: 18px;
	text-align: center;
	margin-bottom: 1rem
}

div.scrollable {
	overflow-x: auto;
	overflow-y: hidden;
}

div.summary-box,
div.content {
	margin: 0 0 0.2rem 0;
	padding: 0;
	border: 1px solid #e7e6e6;
}

div.summary-box {
	line-height: 1.3;
}

div.list {
	padding: 0.2rem 0.5rem;
}

div.captioned_image {
	display: flex;
	flex: 1 1 40%;
	align-items: flex-end;
	gap: 5px;
}

div.captioned_image p.caption {
	font-size: 0.8rem;
	font-style: italic;
	max-width: 150px;
}

div.captioned_image img {
	display: block;
}

.summary-box.clinical {
	background: #ffe9a6;
}

.summary-box.public_health {
	background: #b6cfe6;
}

.summary-box.technical {
	background: #dfdfdf;
}

div.typing {
	margin-top: 0.5rem
}

div.st {
	margin-top: 1rem;
	border: 0;
	line-height: 1.3;
}

div.profile {
	margin-top: 1rem;
	border: 0
}

div.profile td.profile_heading {
	font-size: 16px;
	padding-right: 0.5rem;
	border-bottom: 0 !important;
}

div.profile table {
	table-layout: fixed;
}

div.profile table tr th {
	text-align: center;
	font-size: 0.8rem;
	font-weight: 300;
	border-top: 1px solid #d5d7d9;
	border-bottom: 1px solid #d5d7d9;
	width: 2.5rem;
}

div.profile table tr td {
	text-align: center;
	font-size: 0.8rem;
	border-bottom: 1px solid #d5d7d9;
}

div.list>item {
	display: block;
	margin: 0.2rem 0;
}

div.list>item>label {
	font-weight: bold;
}

div.list>item>label:after {
	content: ":\00A0";
}

value.description {
	font-size: 0.9rem
}

p {
	padding-left: 0.5rem;
}

p.note,
span.note,
ul.note {
	font-size: 0.8rem
}

span.key {
	font-family: monospace;
	font-weight: 600;
	background: #f3f4f5;
	color: #ba4083;
	border: 1px solid #d1d5da;
	padding: 0 0.2rem;
}

table.amr {
	width: 98%;
	margin: auto;
	text-align: left;
	border: 1px dashed #afafaf;
	border-collapse: collapse;
}

table.amr th,
table.amr td {
	font-size: 0.8rem;
	border: 1px dashed #afafaf;
	padding: 5px;
}

table.isolates {
	font-size: 0.8rem;
}

table.isolates tr.this_isolate td {
	background: #ddd
}

p.cluster_summary {
	width: 98%;
	margin: auto;
	font-size: 0.8rem;
}

p.heading {
	font-size: 0.9rem;
	font-weight: 600;
	margin: 0;
}

label.discrete {
	font-weight: 300 !important;
}

value.critical {
	color: #f00
}

table.qc td.label {
	padding-right: 4rem;
}

@media (max-width: 600px) {
	.info-grid {
		display: block
	}
	.info-grid div:not(:first-child) {
		margin-top: 12px;
	}

}
</style>
<title>Technical Genome Report: id-1492</title>
</head>

<body>
  
  
  <section class="report">
  <h1 class="technical">TECHNICAL GENOME REPORT</h1>
  <div class="info-grid">
    <div class="list bordered">     
      	<item><label>Genome identifier</label><value>id-1492</value></item>
      	<item><label>Isolate/laboratory identifier</label><value>M54</value></item>
      	<item><label>Year of isolation</label><value>2020</value></item>
      	<item><label>Location</label><value>Vadodara, India</value></item>   
      	<item><label>Source type</label><value>Human</value></item>     
        <item><label>Host tissue sampled</label><value>Urine</value></item>
    </div>
    <div class="list bordered">   
    	<item><label>Accession number</label><value>Not defined</value></item>
    	<item><label>Genome report date</label><value>2026-01-14</value></item>
    	<item><label>Sequencing technology</label><value>Illumina</value></item>
    	<item><label>Genome quality</label><value>Acceptable
    			 (detailed below)</value></item>
    	<item><label>Version control</label><value></value>Kleborate v3.2.4 analysis on 2025-11-12</item>
    </div>
  </div>
    
	<div class="summary-box technical list">  
	    <item><label>Summary</label>
	    <value><em>Klebsiella pneumoniae</em> with intrinsic ampicillin resistance. Note multidrug resistance - ESBL and CPE producing. One virulence factor present (yersiniabactin) and 7 plasmid types (Col(pHAD28), Col440II, ColKP3, IncFIA(HI1), IncFII, IncR, IncX3).</value></item>  
	</div>
  
    <div class="summary-box organism list">
		<item><label>Organism</label><value><em>Klebsiella pneumoniae</em>. 
		<em>Species match: strong</em></value></item>
    </div>
    
    <div class="content amr">
	
		<div class="list">
		<item><label>Antibiotic resistance determinants</label>
		<value class="description">Resistance determinants identified through Kleborate v3.2.4 analysis on 2025-11-12. 
		Only determinants with >90% coverage and >90% identity shown, also those with imprecise protein matches 
		or reduced coverage will be flagged. Genetic variants (homologs) are highlighted *. "None found" means 
		no resistance mechanisms have been identified in the antimicrobial resistance databases that exist at 
		the time of reporting, correlate with phenotype. &dagger; Macrolides and tetracycline lack breakpoints as not 
		recommended for therapy, resistance determinants included here.   It is best practice to review these 
		with the phenotypic antibiotic susceptibility profile. 
		<a href="https://github.com/klebgenomics/Kleborate" target="_blank">
		<span style="white-space:nowrap">https://github.com/klebgenomics/Kleborate</span></a>.</value></item>
		</div>
		
		<div class="scrollable">
		<table class="amr">
		<tr><th>Drug class</th><th>Acquired genotypes</th></tr>
		<tr><td>Penicillins (expected resistance due to blaSHV)</td><td>blaSHV-11^, blaCTX-M-15, blaNDM-5;OXA-181</td></tr>
		<tr><td>  + &beta;-lactamase inhibitor</td><td>None found</td></tr>
		<tr><td>Cephalosporins (3<sup>rd</sup> gen)</td><td>blaCTX-M-15, blaNDM-5;OXA-181</td></tr>
		<tr><td>  + &beta;-lactamase inhibitor</td><td>None found</td></tr>
		<tr><td>Carbapenems</td><td>blaNDM-5;OXA-181</td></tr>
		<tr><td>Porin mutations (multiple drug classes)</td><td>OmpK36:p.136_137insThrAsp;OmpK35:p.Ile11fs</td></tr>
		<tr><td>Aminoglycosides</td><td>aac(3)-IId^;aadA*;aadA2^;rmtB</td></tr>
		<tr><td>Fluoroquinolones</td><td>qnrS1, GyrA:p.Ser83Phe;GyrA:p.Asp87Asn;ParC:p.Glu84Lys</td></tr>
		<tr><td>Fosfomycin</td><td>None found</td></tr>
		<tr><td>Phenicols</td><td>cmlA5</td></tr>
		<tr><td>Polymixins</td><td>None found</td></tr>
		<tr><td>Tigecycline</td><td>None found</td></tr>
		<tr><td>Trimethoprim</td><td>dfrA12</td></tr>
		<tr><td>  + Sulfonamides</td><td>sul1</td></tr>
		<tr><td>&dagger;Macrolides</td><td>ereA2*;mphA*;ermB.v1*</td></tr>
		<tr><td>&dagger;Tetracycline</td><td>None found</td></tr>
		</table></div>
	
	</div>
  
<div class="content typing">
<div class="list">
<item><label>Bacterial typing</label><value class="description">
subspecies characterisation and classification to assess isolate similarity</value>
</item></div>
<div class="info-grid">
<div class="st">
MLST (Multilocus sequence typing)<br />
Sequence type (ST): 
16
</div>

	<div class="profile">
	<table><tr><td rowspan="2" class="profile_heading">Profile:</td><th>gapA</th><th>infB</th><th>mdh</th><th>pgi</th><th>phoE</th><th>rpoB</th><th>tonB</th></tr><tr><td>2</td><td>1</td><td>2</td><td>1</td><td>4</td><td>4</td><td>4</td></tr></table>
	</div>

</div>

<div class="info-grid">
<div class="list"><item><label class="discrete">Phylogroup</label>
<value>Kp1</value></item></div>
<div class="list"><item><label class="discrete">Sublineage</label>
<value>SL17</value></item></div>
<div class="list"><item><label class="discrete">Clonal group</label>
<value>CG16</value></item></div>
<div class="list"><item><label class="discrete">cgST</label>
<value>12601</value></item></div>
</div>
<div class="list">
<item><label class="discrete">LIN (Life Identification Number)</label><value>0_0_22_27_0_7_0_0_0_0</value></item>
</div>

</div>

<div class="content capsule">

<div class="list"><item><label>Capsule and O typing</label><value class="description">
Polysaccharide K and lipopolysaccharide O serotypes as predicted by KL and O genotype 
<a href="https://github.com/klebgenomics/Kaptive" target="_blank"><span style="white-space: nowrap">
https://github.com/klebgenomics/Kaptive</span></a>.</value></item></div>
<div class="info-grid">
<div class="list">
<item><label class="discrete"><em>K type</em></label><value><em>K81</em></value></item>
<item><label class="discrete">Confidence</label><value>Typeable</value></item>


	<item><label class="discrete">Problems</label><value>?3</value></item>

</div>
<div class="list">
<item><label class="discrete"><em>O type</em></label><value><em>O13</em></value></item>
<item><label class="discrete">Confidence</label><value>Typeable</value></item>


	<item><label class="discrete">Problems</label><value>?3!</value></item>

</div>
</div>

<h3>Problems key:</h3>
<ul class="note">
<li><span class="key">?</span> = the locus was in multiple pieces, possibly due to a poor match or discontiguous assembly. 
The number of pieces is indicated by the integer directly following the ? symbol.</li>
<li><span class="key">-</span> = genes expected in the locus were not found.</li>
<li><span class="key">+</span> = extra genes were found in the locus.</li>
<li><span class="key">*</span> = one or more expected genes was found but with translated identity below the minimum threshold.</li>
<li><span class="key">!</span> = one or more genes was found but truncated.</li>
</ul>


</div>

<div class="content virulence">

<div class="list">
<item><label>Virulence factors</label><value class="description">Factors that may lead to increased 
ability to cause invasive disease <a href="https://github.com/klebgenomics/Kleborate" target="_blank">
<span style="white-space:nowrap">https://github.com/klebgenomics/Kleborate</span></a>.</value></item>
<div class="list">
<item><label class="discrete">Virulence score</label><value>1</value></item>
</div>

<div class="info-grid">
<div class="list">
<item><label>Acquired siderophores</label></item>
<item><label class="discrete">Aerobactin ST</label><value>None</value></item>
<item><label class="discrete">Salmochelin ST</label><value>None</value></item>
<item><label class="discrete">Yersiniabactin ST</label><value>384-2LV</value></item>
</div>
<div class="list">
<item><label>Other factors</label></item>
<item><label class="discrete">Colibactin ST</label><value>None</value></item>
<item><label class="discrete">Hypermucoidy</label><value>None</value></item>
</div></div>
</div>

</div>

<div class="content plasmid">

<div class="list">
<item><label>Plasmid Inc typing</label><value class="description">
 These are identified using PlasmidFinder 
<a href="https://cge.food.dtu.dk/services/PlasmidFinder/" target="_blank">
<span style="white-space: nowrap">https://cge.food.dtu.dk/services/PlasmidFinder/</span></a></value>
</item>
<item><label class="discrete">Plasmid type</label><value>Col(pHAD28), Col440II, ColKP3, IncFIA(HI1), IncFII, IncR, IncX3</value></item>
</div>

</div>

<div class="content outbreak">
<div class="list">
<item><label>Outbreak analysis</label><value class="description">Identifying isolates that are genetically 
similar and maybe part of a transmission chain 
<a href="https://bigsdb.pasteur.fr/klebsiella/cgmlst-lincodes/" target="_blank">
<span style="white-space: nowrap">https://bigsdb.pasteur.fr/klebsiella/cgmlst-lincodes/</span></a>.
</value></item>
</div>

<p class="heading">Allele based:</p>
<div class="list">
<item><label class="discrete">LIN (Life Identification Number)</label><value>0_0_22_27_0_7_0_0_0_0</value></item>
<p class="note" style="margin-top:0">(based on cgMLST scheme with 629 genes)</p>
</div>

<p class="cluster_summary">There are 4 related isolates identified with the same LINcode prefix <a href="https://bigsdb.pasteur.fr/cgi-bin/bigsdb/bigsdb.pl?db=pubmlst_klebsiella_isolates&page=query&designation_field1=lin_18&designation_operator1=starts%20with&designation_value1=0_0_22_27_0&submit=1">0_0_22_27_0</a> (representing the 10 core genome MLST distance threshold) in the database.</p>
<div class="info-grid">
<div class="scrollable">
<table class="isolates"><tr><th>id</th><th>isolate</th><th>country</th><th>year</th><th>LINcode</th></tr><tr><td><a href="https://bigsdb.pasteur.fr/cgi-bin/bigsdb/bigsdb.pl?db=pubmlst_klebsiella_isolates&page=info&id=22874" target="_blank">22874</a></td><td>NMI4821/11</td><td>Poland</td><td>2011</td><td>0_0_22_27_0_0_5_0_0_0</td></tr><tr><td><a href="https://bigsdb.pasteur.fr/cgi-bin/bigsdb/bigsdb.pl?db=pubmlst_klebsiella_isolates&page=info&id=22876" target="_blank">22876</a></td><td>NMI10734/11</td><td>Poland</td><td>2011</td><td>0_0_22_27_0_0_5_0_0_0</td></tr><tr><td><a href="https://bigsdb.pasteur.fr/cgi-bin/bigsdb/bigsdb.pl?db=pubmlst_klebsiella_isolates&page=info&id=22877" target="_blank">22877</a></td><td>NMI10898/11</td><td>Poland</td><td>2011</td><td>0_0_22_27_0_0_5_0_0_0</td></tr><tr><td><a href="https://bigsdb.pasteur.fr/cgi-bin/bigsdb/bigsdb.pl?db=pubmlst_klebsiella_isolates&page=info&id=1495" target="_blank">1495</a></td><td>MyNCGM268</td><td>Myanmar</td><td>2016</td><td>0_0_22_27_0_6_0_0_0_0</td></tr><tr class="this_isolate"><td><a href="https://bigsdb.pasteur.fr/cgi-bin/bigsdb/bigsdb.pl?db=pubmlst_klebsiella_isolates&page=info&id=1492" target="_blank">1492</a></td><td>M54</td><td>India</td><td>2020</td><td>0_0_22_27_0_7_0_0_0_0</td></tr></table>
</div>

<p class="note">Most isolates with identical LIN codes (all 10 thresholds) have <100 and often <25 single nucleotide 
differences, but further single nucleotide analysis may be required for these isolates in cluster/outbreak 
investigations.</p> 

</div>


</div>

<div class="content qc">
<div class="list">
<item><label>QC status</label><value class="description">based on KlebNet criteria.
</value></item>

<item><label>Genome quality</label><value>Acceptable</value></item>
<table class="qc">
<tr><td class="label">Species match:</td>
<td>strong <span class="note">[determined with Kleborate]</span></td></tr>
<tr><td>Genome size:</td>
<td>5,567,000 bp <span class="note">[accepted criteria 4,969,898; 6,132,846]</span></td></tr>
<tr><td>Contigs:</td>
<td>150 <span class="note">[accepted criteria <500]</span></td></tr>
<tr><td>%GC content:</td>
<td>57.23 <span class="note">[accepted criteria 56.35-57.98]</span></td></tr>
</table>
</div>
</div>

  </section>
</body>
</html>
