<!DOCTYPE html>
<head>
<style>
@page {
    size: A4;
    margin: 8px;
}
body {font: 1em Arial,sans-serif;margin: 0 8px;background:#ede9de}
h1 {font-size:1.2em;line-height:120%;font-weight:400;padding:0.2em;border:1px solid black}
h2 {font-size:1.1em;font-weight:400;margin-top:0.2em}
.report_title {margin:0.5em 0 0 1.5em;padding:0}
.report_title p {background:#c0c0c0}
div.section {display:flex;justify-content:space-between;margin-bottom:1em;column-gap:10px}
div.block {border:1px solid #a6a6a6;padding:0.5em;margin-bottom:1em}
th {background:#fff;text-align:center}
td {background:#fff;min-width:3em;text-align:center}
table.amr {border-spacing:0;border-collapse:collapse}

table.amr td {background:transparent;border:1px solid #888;}
</style>
</head>
<html>
<body>
<div class="section">
<h1 style="max-width:60%">KLEBSIELLA SPECIES GENOME SEQUENCE REPORT</h1>
<div class="report_title"><p>Level 1: Clinical/microbiology use</p></div>
</div>

<div class="section">
  <div style="border:1px solid navy;padding:0.5em">

    
<strong>Database identifier: </strong>9439<br />
<strong>Isolate identifier: </strong>CriePir26<br />

<strong>Alternative identifier(s): </strong>


<br />

<strong>Date entered: </strong>2019-08-06<br />
<strong>Location: </strong>

Moscow,

Russia<br />

</dl>
</div>

<div style="border:1px solid orange;padding:0.5em">

<strong>Genome report date: </strong>2023-12-15<br />
<strong>Genome quality: </strong>
 Marginal
</div>
</div>













<div class="block" style="border:2px solid #ffcf3c;background:#ffe69c">
<strong>Summary: </strong>
This isolate is <em>Klebsiella pneumoniae</em>, demonstrating genotypic resistance determinants to aminoglycocides, carbapenems, 3rd generation cephalosporins, fluoroquinolones, phenicols, sulfonamides and trimethoprim, with intrinsic ampicillin resistance.

</div>
<div class="block">
<strong>Organism: </strong><em>Klebsiella pneumoniae</em><br />
</div>
<div class="block">
<p><strong>Bacterial typing: </strong> subspecies classifications to assess isolate similarity</p>
<div class="section">
<div>
MLST (Multilocus sequence typing)<br />
Sequence type (ST): 377
</div>
<div>

Profile:
<table><tr><th>gapA</th><th>infB</th><th>mdh</th><th>pgi</th><th>phoE</th><th>rpoB</th><th>tonB</th></tr><tr>
<td>10</td><td>20</td><td>2</td><td>1</td><td>9</td><td>11</td><td>12</td>
</tr>
</table>
</div>
</div>
</div>
<div class="block">
<p><strong>Antibiotic susceptibility: </strong>Resistance mechanisms identified are used to predict antimicrobial susceptibility.</p>

<table class="amr">
<tr><td><strong>Drug class</strong></td><td><strong>Resistance determinants</strong></td><td><strong>Drug class</strong></td><td><strong>Resistance determinants</strong></td></tr>
<tr>
<td>Aminoglycocides</td><td>aac(3)-IIa;aac(6')-Ib-cr;aadA;aph(3')-VIa (homolog);strA</td>


<td>Penicillins</td><td>OXA-1,SHV-110</td></tr>


<tr><td>Carbapenems</td><td>OXA-48,OmpK35-90%;OmpK36GD</td>

<td>Penicillins + β-lactamase inhibitors</td><td></td></tr>
<tr><td>Cephalosporins (3rd gen)</td><td>CTX-M-15</td>
<td>Phenicols</td><td>CatB4 (fragment);catA1 (homolog)</td></tr>
<tr><td>Cephalosporins (3rd gen.) + β-lactamase inhibitors</td><td></td>
<td>Sulfonamides</td><td>sul1 (homolog, fragment);sul2</td></tr>


<tr><td>Colistin</td><td></td>

<td>Tetracycline</td><td></td></tr>


<tr><td>Fluoroquinolones</td><td>GyrA-83Y;GyrA-87A;ParC-80I</td>

<td>Tigecycline</td><td></td></tr>
<tr><td>Fosfomycin</td><td></td>
<td>Trimethoprim</td><td>dfrA15</td></tr>
</table>

</div>

<div class="block">
<p><strong>Capsular typing: </strong>Polysaccharide K and lipopolysaccharide O types as predicted by KL and O genotype</p>
<p>
<span style="margin-right:100px"><em>K type: unknown (KL102)</em></span>
<span><em>O type: O2afg</em></span></p>

</div>

</body>
</html>

